x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -1137,7 +1137,7 @@ const updateExistingDataStream = async ({
   // Trigger a rollover if the index mode or source type has changed
   if (
     currentIndexMode !== settings?.index?.mode ||
-    currentSourceType !== mappings?._source?.mode ||
+    currentSourceType !== settings?.index?.source?.mode ||
```

Comment:
should we skip rollover when an existing data stream is updated from `mappings._soruce.mode` to `settings.index.mapping.source`?

Reply:
I think we still want to do a rollover in that case here to be sure the write backing  indices is using the correct settings.